test/verifyConfigs.test.ts

```diff
@@ -34,6 +34,11 @@ describe('Verify config files', () => {
       const file = fs.readFileSync(USERS_JSON, 'utf-8');
       assert.doesNotThrow(() => JSON.parse(file));
     });
+ 
+    it('has no @ symbol', () => {
```

Comment:
I think this was a good idea for a test, your idea being to make sure that we don't accidentally add someone's email address rather than their GitHub username?